python/src/test/scala/latis/AnomalyDataSpec.scala

```diff
@@ -20,40 +20,68 @@ class AnomalyDataSpec extends FlatSpec {
     }
   }
 
-  "The anomalous sine wave dataset" should "be manipulated by a rolling mean script" in {
+  "The rolling mean script" should "manipulate the anomalous sine wave dataset" in {
     val ds = Dataset.fromName("sine_wave_with_anomalies")
       .withOperations(Seq(Selection("time", ">=" , "2000-01-01"), 
         DetectAnomaliesWithRollingMean(dsName="SineWave")))
 
     //TextWriter().write(ds)
 
     StreamUtils.unsafeHead(ds.samples) match {
-      //TODO: Need a Data trait to match on Booleans?
-      case Sample(DomainData(Number(t)), RangeData(Real(f), Real(rm), o: Data.BooleanValue)) =>
+      case Sample(DomainData(Number(t)), RangeData(Real(f), Real(rm), BooleanDatum(o))) =>
         t should be (1)
         f should be (0.841470985)
         rm should be (0.9432600027000001)
-        o.value should be (false)
+        o should be (false)
     }
+  }
+
+  "The rolling mean script" should "also work with Text times" in {
```

Comment:
```suggestion
  it should "also work with Text times" in {
```
You can use the cool "it" syntax here